index.d.ts

```diff
@@ -133,6 +133,38 @@ type ReaderRevenuePosition =
     | 'ampHeader'
     | 'ampFooter';
 
+interface CampaignsType {
+    id: string;
+    name: string;
+    rules: [];
+    priority: number;
```

Comment:
Is there a more defined list of possible numbers? Like DefCon 1-5?

Reply:
again not sure,  will investigate further

Reply:
Lets remove any properties that we aren't using